add-on/chrome/bootstrap.js

```diff
@@ -483,6 +489,7 @@ var WindowListener = {
         gBrowser.removeEventListener("mousemove", this);
         this._listeningToTabSelect = false;
         this._browserSharePaused = false;
+        this._constants = null;
```

Comment:
Please move this code to a lazy getter like `get toolbarbutton()` and `get panel()` and `get browser()`, so that you only retrieve these constants once from MozLoopAPI.